README.md

```diff
@@ -165,47 +165,13 @@ performance:
 
 ## Procedure
 
-### 1) Install the MinIO Operator
-
-Run the following commands to install the MinIO Operator and Plugin using the Kubernetes ``krew`` plugin manager:
-
-```sh
-kubectl krew update
-kubectl krew install minio
-```
-
-See the ``krew`` [installation documentation](https://krew.sigs.k8s.io/docs/user-guide/setup/install/) for instructions
-on installing ``krew``.
-
-Run the following command to verify installation of the plugin:
-
-```sh
-kubectl minio version
-```
-
-As an alternative to `krew`, you can download the `kubectl-minio` plugin from
-the [Operator Releases Page](https://github.com/minio/operator/releases). Download the `kubectl-minio` package
-appropriate for your operating system and extract the contents as `kubectl-minio`. Set the `kubectl-minio` binary to be
-executable (e.g. `chmod +x`) and place it in your system `PATH`.
-
-For example, the following code downloads the latest stable version of the MinIO Kubernetes Plugin and installs it to
-the system ``$PATH``. The example assumes a Linux operating system:
+### 1) Install the MinIO Operator via Kustomization
 
```

Comment:
https://kubectl.docs.kubernetes.io/installation/kustomize/

Should we add a link about the above in here, since this is distinct functionality from what `kubectl kustomize` supports? Spitballing:


```suggestion
Follow the  [Install `kustomize`](https://kubectl.docs.kubernetes.io/installation/kustomize/)  guide for your host system before starting this procedure.

```